frontend/src/scenes/sessionRecordings/sessionRecordingsTableLogic.test.tsx

```diff
@@ -0,0 +1,71 @@
+import { sessionRecordingsTableLogic } from './sessionRecordingsTableLogic'
+import { sessionRecordingsTableLogicType } from './sessionRecordingsTableLogicType'
+import { BuiltLogic } from 'kea'
+import { mockAPI } from 'lib/api.mock'
+import { expectLogic, initKeaTestLogic } from '~/test/kea-test-utils'
+
+jest.mock('lib/api')
+
+describe('sessionRecordingsTableLogic', () => {
+    mockAPI(async ({ pathname, searchParams }) => {
+        if (pathname === 'api/projects/@current/session_recordings' && searchParams['distinct_id'] === '') {
+            return {
+                results: ['List of recordings from server'],
+            }
+        } else if (
+            pathname === 'api/projects/@current/session_recordings' &&
+            searchParams['distinct_id'] === 'cool_user_99'
+        ) {
+            return {
+                results: ["List of specific user's recordings from server"],
+            }
+        } else {
+            throw new Error(`Unmocked fetch to: ${pathname} with params: ${JSON.stringify(searchParams)}`)
+        }
+    })
+
+    describe('global logic', () => {
+        let globalLogic: BuiltLogic<sessionRecordingsTableLogicType<string>>
```

Comment:
I think this could just be one `logic` defined higher up, instead of having two differently named logics that will never meet.